dev-aws/kafka-shared-msk/customer-proposition/customer-proposition.tf

```diff
@@ -38,6 +38,35 @@ resource "kafka_topic" "uswitch_events_v1" {
   }
 }
 
+resource "kafka_topic" "uswitch_events_v2" {
+  name = "customer-proposition.uswitch.events.v2"
+
+  replication_factor = 3
+  partitions         = 15
```

Comment:
Is this a migrated topic? If not, can it do with less partitions?

Reply:
Potentially. We are migrating from v1 to v2, but do plan to cut down the number of events in v2. 
Do you have a suggestion on how many partitions is preferred?

Reply:
It boils down to how many parallel consumers you need, depending on the throughput needs of your consumers.
If you're uncertain, just go with 5, as you can scale it up afterwards

Reply:
We can give 5 a go. That's done now 👍🏻